src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/RequestFeeRemissionAipPreparerTest.java

```diff
@@ -227,6 +209,7 @@ void handle_should_append_previous_remission_details(
         when(asylumCase.read(APPEAL_TYPE, AppealType.class)).thenReturn(Optional.of(appealType));
         when(asylumCase.read(REMISSION_OPTION, RemissionOption.class)).thenReturn(Optional.of(previousRemissionOptionOption));
         when(asylumCase.read(PREVIOUS_REMISSION_DETAILS)).thenReturn(Optional.of(previousRemissionDetails));
+        LocalDate currentDate = LocalDate.now();
 
```

Comment:
Can you verify asylumCase.write(AsylumCaseFieldDefinition.REQUEST_FEE_REMISSION_DATE) is completed once?